KsApi/models/graphql/GraphMutationPostCommentResponseEnvelopeTests.swift

```diff
@@ -0,0 +1,29 @@
+@testable import KsApi
+import XCTest
+
+final class GraphMutationPostCommentResponseEnvelopeTests: XCTestCase {
+  func testDecode() {
+    let dictionary: [String: Any] = [
+      "createComment": [
+        "comment": [
+          "body": "Hello World",
+          "id": "Q29tbWVudC0zMjY2MjU0MQ=="
+        ]
+      ]
+    ]
+
+    guard let data = try? JSONSerialization.data(withJSONObject: dictionary, options: []) else {
+      XCTFail("Should have data")
+      return
+    }
+
+    do {
+      let envelope = try JSONDecoder().decode(GraphMutationPostCommentResponseEnvelope.self, from: data)
+      XCTAssertEqual(envelope.createComment.comment.body, "Hello World")
```

Comment:
Not a fan of calling `envelope.createComment.comment.body` but I don't think we can avoid it, given the structure of the response?

Reply:
Do you need the `CreateComment` struct? Maybe it can be `envelope.comment.body`?

Reply:
Oh wait, I guess you do because the GQL query has a `createComment` object being returned.

Reply:
Yep, I'm relying on Decodable for parsing the response here.

Reply:
You could also override the initializer with an `init(from decoder: Decoder) { ... }` and do some manual drilling down through the nested containers but could go either way.

Reply:
Updated this.